tests/network-ovn

```diff
@@ -48,8 +48,6 @@ setupCloudInitReady() {
 
   cat <<EOF | lxc profile set "${profile}" --project "${instProj}" cloud-init.user-data -
 #cloud-config
-packages:
-  - curl
```

Comment:
I originally needed this on the `network` scripts because the sriov part of the script needs `images:ubuntu/22.04` so it has the relevant intel nic drivers to work.

Reply:
With the move to `ubuntu:` and possibly `ubuntu-minimal:` we can assume `curl` to be present so not having it listed there saves the time to run `apt-get update`. 

Considering that tests would fail in an obvious way if for some reason `curl` would be missing, are you OK with keeping it that way?

Reply:
Yeah im fine with it being removed here, just a note that this approach wont work for all tests :)

Reply:
Eventually, I'd like to migrate from `ubuntu:` to a local image based on a tweaked `ubuntu-minimal:` image, exactly as done in this PR. In such case, if a package would be missing we could decide to either bake it inside the base image or install it just for the test needing it.